app/main.dev.js

```diff
@@ -82,8 +82,24 @@ const handleSlippiURIAsync = async (url) => {
   // Specifying a base will provide sane defaults if the input is null or wrong
   const myUrl = new URL(url, `${slippiProtocol}://null`);
   log.info(`protocol: ${myUrl.protocol}, hostname: ${myUrl.hostname}`);
-  if (myUrl.protocol !== `${slippiProtocol}:`) {
-    return;
+  if (myUrl.protocol !== `${slippiProtocol}:` && fs.existsSync(url)) {
+    const wait = ms => new Promise((resolve) => setTimeout(resolve, ms));
```

Comment:
This looks like duplicate logic, would be nice if we could figure out a clean way to DRY it